src/index.js

```diff
@@ -23,39 +23,29 @@ bot.on('ready', () => {
   console.log(`Logged in as ${bot.user.tag}!`);
 });
 
-// Testing objects for enemy and player
-let enemy;
-
-let player = {
-  name: 'Hero',
-  hp: 100,
-  atk: 20,
-  def: 7,
+const ctx = {
+  player: {
+    name: 'Hero',
+    hp: 100,
+    atk: 50,
+    def: 7,
+  },
+  enemy: null,
+  inventory: [],
 };
 
-let inventory = [];
-
 const TOWN_COMMANDS = {
-  inventory: () => checkInv(inventory),
-
   greet,
 
-  hunt: () => {
-    enemy = hunt();
-    return enemy.msgs.encounter;
-  },
+  inventory: () => checkInv(ctx),
+
+  hunt: () => hunt(ctx),
```

Comment:
You can actually just put these like this and call it later as `commandHandler(ctx)`, I think that will work better going forwards - try and make our global variable look as not-global as possible for when we finally get to make it not global

```suggestion
  inventory: checkInv,

  hunt,
```